docs/src/ref_impl/cross_compilation.md

```diff
@@ -5,15 +5,16 @@
 
 # Cross-Compilation
 
-Ghaf is targeted at a range of devices and form factors that support different instruction set architectures (ISA). Many small form-factor edge devices are not powerful enough to compile the needed applications or OSs that run on them. As the most common ISA used in desktops and servers is ``x\_86``, this will generally require that the code is cross-compiled for target ISA e.g. ``AArch64`` or ``RISC-V``.
+Ghaf is targeted at a range of devices and form factors that support different instruction set architectures (ISA). Many small form-factor edge devices are not powerful enough to compile the needed applications or OSs that run on them. As the most common ISA used in desktops and servers is ``x_86``, this will generally require that the code is cross-compiled for target ISA e.g. ``AArch64`` or ``RISC-V``.
 
 NixOS and Nixpkgs have good support for cross-compilation, however, there are still some that can not be compiled in this way.
 
+
 ## binfmt
```

Comment:
@vilvo Is `binfmt` still a thing to support? If yes, we need to provide real example of how to use it (I admit I could not get it working)

Reply:
I agree that we should either drop the support or link to known issues with it. I recommend dropping as I don't think @cazfi et al. are working on it.

Reply:
Let's add know issues. @unbel13ver will help.